workshops/intro_to_ML/README.md

```diff
@@ -0,0 +1,272 @@
+    Epoch 3/700
+    1/1 [==============================] - 0s 1ms/step - loss: 3.8004
+    Epoch 4/700
+    1/1 [==============================] - 0s 1ms/step - loss: 2.6229
+    Epoch 5/700
+    1/1 [==============================] - 0s 1ms/step - loss: 1.8128
+    Epoch 6/700
+    1/1 [==============================] - 0s 1ms/step - loss: 1.2554
+    Epoch 7/700
+    1/1 [==============================] - 0s 1ms/step - loss: 0.8718
+    Epoch 8/700
+    1/1 [==============================] - 0s 1ms/step - loss: 0.6078
+    Epoch 9/700
+    1/1 [==============================] - 0s 1ms/step - loss: 0.4260
+    Epoch 10/700
+    1/1 [==============================] - 0s 1ms/step - loss: 0.3008
+    .
+    .
+    .
+    .
+    .
+    .
+    .
+    .
+    Epoch 690/700
+    1/1 [==============================] - 0s 4ms/step - loss: 7.1252e-08
+    Epoch 691/700
+    1/1 [==============================] - 0s 1ms/step - loss: 6.9955e-08
+    Epoch 692/700
+    1/1 [==============================] - 0s 1ms/step - loss: 6.8631e-08
+    Epoch 693/700
+    1/1 [==============================] - 0s 1ms/step - loss: 6.7373e-08
+    Epoch 694/700
+    1/1 [==============================] - 0s 1ms/step - loss: 6.6085e-08
+    Epoch 695/700
+    1/1 [==============================] - 0s 1ms/step - loss: 6.4882e-08
+    Epoch 696/700
+    1/1 [==============================] - 0s 1ms/step - loss: 6.3664e-08
+    Epoch 697/700
+    1/1 [==============================] - 0s 1ms/step - loss: 6.2483e-08
+    Epoch 698/700
+    1/1 [==============================] - 0s 1ms/step - loss: 6.1313e-08
+    Epoch 699/700
+    1/1 [==============================] - 0s 1ms/step - loss: 6.0135e-08
+    Epoch 700/700
+    1/1 [==============================] - 0s 1ms/step - loss: 5.8995e-08
+
+    <tensorflow.python.keras.callbacks.History at 0x7f24d74b2470> 
+```
+<br/>
+
+
+# Prediction
+Ok, now we have a model that has been trained to learn the relationship between X and Y. We can use the **model.predict** method to have it figure out the Y for a previously unknown X. So, for example, if X = 10, what do you think Y would be?   
+Lets run the code to see
+
+```py
+print(model.predict([10.0]))
+```
+    [[21.00056]]
+    
+
+
+You might have thought 21, right? But it ended up being a little over. Why do you think that is?  
+
+Remember that the neural networks deal with probabilities, so given the data that we fed the Neural network with, it calculated that there is a very high probability that the relationship between X and Y is Y=2x+1, but with only 6 data values we can't know for sure. As a result, the value for 10 is very close to 21, but not necessarily 21.  
+
+As We work with neural networks, we'll see this pattern recurring. We will almost always deal with probabilities, not certainties, and will do a little bit of coding to figure out what the result is based on the probabilities, particularly when it comes to classification.  
+
+
+<br/>
```

Comment:
what's with all this spacing?

Reply:
It does not reflect in the output. I just added it to make it more readable

Reply:
can you remove it please, it just looks off